src/Raven.Client/Documents/Operations/AI/EmbeddingsTransformation.cs

```diff
@@ -12,10 +13,18 @@ public class EmbeddingsTransformation
 
     public ChunkingOptions ChunkingOptions { get; set; } = new() { ChunkingMethod = ChunkingMethod.PlainTextSplit, MaxTokensPerChunk = 256 };
 
-    public bool ValidateScript()
+    public void Validate(List<string> errors)
```

Comment:
While this method is for internal usage only, I'm concerned about backward compatibility since it's `public`